src/Features/OnboardingTasks.php

```diff
@@ -73,6 +74,23 @@ public function add_media_scripts() {
 		wp_enqueue_media();
 	}
 
+	/**
+	 * Gets an array of plugins that can be installed & activated via the onboarding task list.
+	 *
+	 * @param array $plugins Array of plugin slugs to be allowed.
+	 *
+	 * @return array
+	 */
+	public function get_onboarding_tasks_allowed_plugins( $plugins ) {
```

Comment:
Is there a reason why you added this here vs the `get_onboarding_allowed_plugins` function in the Onboarding.php file.

Reply:
@louwie17 good question. We whitelist plugins in multiple places, I figured OnboardingTasks is the feature that encapsulates the setup tasks? Honestly I'm not 100% on that though.

Reply:
Gotcha, no worries then, although the function I did reference whitelists most of the onboarding payment plugins.

Reply:
@louwie17 good point. I'll move this, quite happy for it all to be in one place anyway